docs/getting-started/rbac/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tekton-triggers-example-clusterbinding
```

Comment:
small nit: maybe we can call this one tekton-triggers-getting-started-binding or something like that?